src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudTest.java

```diff
@@ -49,6 +48,32 @@
 @RunWith(MockitoJUnitRunner.class)
 public class ComputeEngineCloudTest {
 
+  public static final PrivateKey PRIVATE_KEY;
```

Comment:
Do these need to be public? Also, please remove the extra '\n' on line 50.

Reply:
No, they were public when these values were shared between ClientFactoryTest.java and this file, now that they are isolated to this file I've fixed that. And removed the extra '\n'